api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -185,6 +185,9 @@ message ConnectionMetadata {
 
   // Protocol specifies protocol that was captured
   string Protocol = 3 [(gogoproto.jsontag) = "proto,omitempty"];
+
+  // ConnectorID is the ID of the connector used
+  string ConnectorID = 4 [(gogoproto.jsontag) = "connector_id,omitempty"];
```

Comment:
I don't think this is the right place for this value.

The auth connector is only relevant for login events (and maybe IdP MFA events), but `ConnectionMetadata` is used in all sorts of events.

Reply:
Does here make more sense?

Reply:
Have moved the field to userLogin